dali/operators/numba_function/numba_func.cc

```diff
@@ -48,6 +48,21 @@ If no setup function provided, the output shape and data type will be the same a
 .. note::
     This operator is experimental and its API might change without notice.
 
+.. warning::
+    The ``run_fn`` and ``setup_fn``, when :ref:`conditional mode <conditional_execution>` is enabled,
+    must not be transformed by the AutoGraph. There are two conditions to prevent that:
+
+        1. The ``run_fn`` and ``setup_fn`` must be defined at a global scope
+           (i.e. outside of ``pipeline_def`` scope).
+
+        2. If the ``run_fn`` and ``setup_fn`` is created within pipeline definition,
+           the factory function must be decorated with
```

Comment:
What is the meaning of "factory" function here?

Reply:
Leftover from copying the ES warning. Removed.